index.html

```diff
@@ -54,7 +54,174 @@ <h1>I’m Godfrey
+          no accounts or sign-ups required.</p>
+        <ul class="work_languages">
+         <li>html</li>
+         <li>css</li>
+         <li>javascript</li>
+        </ul>
+          <button class="btn">See Project</button>
+      </div>
+
+  </article>
+
+  <article id="pic4" class="work_content">
+    <img src="./image/Snapshoot Portfolio (4).svg" alt="">
+    <div class="work_description">
+      <h2>Tonic</h2>
+      <ul class="conduct">
+        <li class="conduct_head">CANOPY</li>
+        <li><img src="./image/Counter.svg" alt=""></li>
+        <li class="conduct_para">Back End Dev</li>
+        <li><img src="./image/Counter.svg" alt=""></li>
+        <li class="conduct_year">2015</li>
+      </ul>
+      <p class="work_paragraph">A daily selection of privately personalized reads; 
+        no accounts or sign-ups required.</p>
+      <ul class="work_languages">
+       <li>html</li>
+       <li>css</li>
+       <li>javascript</li>
+      </ul>
+        <button class="btn">See Project</button>
+    </div>
+
+</article>
+    </div>
+</section>
+<section class="about">
+<article class="about_head">
+  <h2>About
+    Myself</h2>
+  <p>Hello I’m a software developer! 
+    I can help you build a product , 
+    feature or website Look through 
+    some of my work and experience! 
+    If you like what you see and have 
+    a project you need 
+    coded, don’t hestiate to contact me.</p>
+  <a href="">LET’S CONNECT</a>
+  <ul>
+    <li class="item1"><a href=""><img src="./image/Vectorgithub.svg" alt=""></a></li>
+    <li class="item1"><a href=""><img src="./image/Linkedin iconlinkedin.svg" alt=""></a></li>
+    <li class="item1"><a href=""><img src="./image/Vectormedium.svg" alt=""></a></li>
+    <li class="item1"><a href=""><img src="./image/Vectortwitter (1).svg" alt=""></a></li>
+    <li class="item1"><a href=""><img src="./image/Vector (3).svg" alt=""></a></li>
+  </ul>
+  <button class="btn">Get my resume</button>
+</article>
+<article class="about_language">
+<div class="item">
+  <h3>Languages</h3>
+  <img src="image/down.svg" alt="">
+</div>
+<ul class="about_lang">
+   <li>
+     <img src="image/javascript.svg" alt="">
+     <p>javascript</p>
+   </li>
+   <li>
+    <img src="image/html (2).svg" alt="">
+    <p>html</p>
+   </li>
+   <li>
+    <img src="image/css.svg" alt="">
+    <p>css</p>
+   </li>
+</ul>
+<li class="item">
+  <h3>Framework</h3>
+  <img src="image/left2.svg" alt="">
+
+</li>
+<hr>
+
+<li class="item">
+  <h3>Skills</h3>
+  <img src="image/left2.svg" alt="">
+</li>
+<hr>
+</article>
+   </section>
+  <section class="contact">
+    <article class="contact_footer">
+      <h1>contact me</h1>
+  <img src="image/contact form shapes-mobile@2x.svg" alt="" class="mobile-img">
+      <p>If you have an application you are interested
+         in developing, a feature that you need built or a project
+          that needs coding. I’d
+         love to help with it</p>
+         <form class="form-start">
```

Comment:
- [ ] You guys are doing a great job 💯 so far but please make sure you're using the `action` attribute with the proper link as the value, you'll get the link after signing up at [Formspree service](https://formspree.io/html) this will integrate your form with Formspree service and allows you to receive emails from your future clients ✔.
    -  It should look something like this: `https://formspree.io/f/{form_id}` the `form_id` is the ID you get after signing up.
- [ ] Also kindly make sure you're using the `method` attribute with a `POST` value as this will encrypt the information in the form which will help keep your client's information safe 💂‍♂️.